### PR TITLE
Add Windows Container Entry to FAQ

### PR DESCRIPTION
**Pull request by levlaz**
I am taking this gem that @jedevc shared in discord the other day and adding it to a FAQ entry since this is a common question that we hear.

https://discord.com/channels/707636530424053791/1242180731548209293/1260175488840634448

### CODE REVIEW

File: docs/current_docs/faq.mdx [hunk 1]
```diff
@@ -137,6 +137,10 @@ No. "Rootless mode" means running the Dagger Engine as a container without the `
 
 Yes. By replacing the `/etc/dagger/engine.toml` file with your custom configuration, you can configure the Dagger Engine with CA certs, registry mirrors or additional privileges. However, note that this is a temporary solution and may change in future. [Read more about Dagger Engine configuration](https://github.com/dagger/dagger/blob/main/core/docs/d7yxc-operator_manual.md).
 
+### Can I use Dagger Engine to build Windows Containers?
+
+Unfortunately, not right now. Dagger runs on top of BuildKit and support for Windows Containers is still experimental in BuildKit. In addition, Dagger has a lot of custom code written on top of BuildKit such as networking, init systems, and Linux namespace entries, that do not have exact parallels on Windows.
```
Comment (sipsma) [inline docs/current_docs/faq.mdx:142]: This used to be in our docs but got lost during the recent module renovation I think: we *do* support some limited operations on Windows container images; namely just purely file operations. It's mainly just `withExec` that doesn't work.

This is covered in our integ tests ([here](https://github.com/dagger/dagger/blob/701705f67f11518b89f4ad4ebeb65a553d733f09/core/integration/platform_test.go#L201)) so I feel comfortable mentioning it if you think that level of detail is worth it.

**Review by sipsma — APPROVED** [review]
One non-blocking suggestion, otherwise LGTM

**Comment by levlaz** [comment]
> 
> ​ This is covered in our integ tests ( here (
> https://github.com/dagger/dagger/blob/701705f67f11518b89f4ad4ebeb65a553d733f09/core/integration/platform_test.go#L201
> ) ) so I feel comfortable mentioning it if you think that level of detail
> is worth it.
> 
> 

This is really cool! I had no idea we supported any aspect of this at all.

I feel like mentioning more details here would give folks some false hope. It seems that in order to get this to work properly we need to prioritize the work to do it and that is not currently planned as far as I know.

On the other hand, I don't think we're opposed from anyone in the community who cares about this to start hacking on it similar to any other thing that we don't currently fully support.

I would want to add more detail if we think its worth adding a #windows channel in discord and pointing people there to discuss all the things that it would take to get this to work.

What do you think?

Lev Lazinskiy
Head of Solutions @ dagger.io ( http://dagger.io )
e: ***@***.***
p: 415.470.2142

**Comment by sipsma** [comment]
> I feel like mentioning more details here would give folks some false hope. It seems that in order to get this to work properly we need to prioritize the work to do it and that is not currently planned as far as I know.

Yeah I felt sort of torn around this too. It's nice that at least file ops work today and that _could_ unblock people for some specific situations, but the full support is not complete by any means (and like you said hard to promise much more deep work in this area due to bandwidth).

I just also remembered that besides file operations, you can generally pull+push windows images too, which has some utility for using dagger to manage registry images. But doesn't significantly change the facts here.

So I'm good with just leaving the docs as is for now and waiting for something more fleshed out to add more detail. If we get more specific questions or hit real-world use cases that benefit from the file operation or registry pull/push support we can revisit and consider adding this.

**Comment by levlaz** [comment]
Sounds good to me. We'll always have this conversation to point to as well :) Thanks you!

Lev Lazinskiy
Head of Solutions @ dagger.io ( http://dagger.io )
e: ***@***.***
p: 415.470.2142

**Pull request merged**